src/content/learn/thinking-in-react.md

```diff
@@ -437,15 +438,21 @@ td {
 
 </Sandpack>
 
-Notice that editing the form doesn't work yet. There is a console error in the sandbox above explaining why:
+Primetićete da promena forme još uvek ne radi. Pojavio se problem u konzoli u sandbox-u iznad koji objašnjava zašto:
 
 <ConsoleBlock level="error">
 
 You provided a \`value\` prop to a form field without an \`onChange\` handler. This will render a read-only field.
 
 </ConsoleBlock>
 
-In the sandbox above, `ProductTable` and `SearchBar` read the `filterText` and `inStockOnly` props to render the table, the input, and the checkbox. For example, here is how `SearchBar` populates the input value:
+<ConsoleBlock level="error">
```

Comment:
I've added new console block to show translated message also. Is this correct way, or we can use something else.

Examples on other languages:
![image](https://github.com/user-attachments/assets/b4167989-484c-4277-a72a-a6980dee43bd)

![image](https://github.com/user-attachments/assets/bd10e7f0-84c7-44fe-9fb4-1e872dcda1d4)

![image](https://github.com/user-attachments/assets/274fbd9b-02e9-4b0e-8dc2-ff5637385ad5)